src/audio/aria/aria.toml

```diff
@@ -0,0 +1,25 @@
+	// # Aria module config
+	[[module.entry]]
+	name = "ARIA"
+	uuid = "99F7166D-372C-43EF-81F6-22007AA15F03"
+	affinity_mask = "0x1"
+	instance_count = "8"
+	domain_types = "0"
+	load_type = "0"
+	init_config = "1"
+	module_type = "30"
+	auto_start = "0"
+	sched_caps = [1, 0x00008000]
+
+	// # pin = [dir, type, sample rate, size, container, channel-cfg]
+	pin = [0, 0, 0xfeef, 0xf, 0xa, 0x45ff,
+			1, 0, 0xfeef, 0xf, 0xa, 0x45ff]
+
+	// # mod_cfg [PAR_0 PAR_1 PAR_2 PAR_3 IS_BYTES CPS IBS OBS MOD_FLAGS CPC OBLS]
+	mod_cfg = [0, 0, 0, 0, 260, 1063000, 16, 21, 0, 0, 0,
```

Comment:
How about this? Unlike affinity-mask and instnace-count, the "mod-cfg" field is more related to module implementation, but this too will vary from build target to another. Would we e.g. use an ifdef-include tree here to specify a specific mod-cfg value (let's say for a Hifi4 optimized Xtensa target) and use generic values for a regular ANSI C build? @lgirdwood you had some thoughts on this

Reply:
This will be dealt with via
```
#if CONFIG_HIFI3
// hifi 3 mod cfg
#elif CONFIG_HIFI4
// hifi4 mod cfg
#elif and so on
```
but this is a future update.